test/promql/tests/data_plane/pipeline_overhead_test.yaml

```
@@ -247,3 +247,50 @@ tests:
     alert_rule_test:
       - eval_time: 85m
         alertname: ChainsControllerPerformance
+
+
+  # ----- PAC Controller Performance / High Overhead ----
+  - interval: 1m
+    input_series:
+
+      # see https://prometheus.io/docs/prometheus/latest/configuration/unit_testing_rules/ for explanations of the expanding notation used for the values
+      - series: 'pac_watcher_client_latency_bucket{le="1", source_cluster="cluster01"}'
+        values: '300+300x780'
+      - series: 'pac_watcher_client_latency_bucket{le="10", source_cluster="cluster01"}'
+        values: '300+300x780'
+      - series: 'pac_watcher_client_latency_bucket{le="+Inf", source_cluster="cluster01"}'
+        values: '900+900x780'
+      - series: 'pac_watcher_client_latency_bucket{le="1", source_cluster="cluster01"}'
+        values: '1x780'
+      - series: 'pac_watcher_client_latency_bucket{le="+Inf", source_cluster="cluster01"}'
```

```suggestion
      - series: 'pac_watcher_client_latency_bucket{le="+Inf", source_cluster="cluster02"}'
```

again labeling nuance